packages/documentation-site/patternfly-docs/pages/featured-posts-data.json

```diff
@@ -0,0 +1,30 @@
+{
+  "post1": {
+    "title": "Accessibility best practices",
+    "author": "Agnieszka Gancarczyk",
+    "length": "8 min read",
+    "URL": "https://medium.com/patternfly/accessibility-best-practices-e98a13bdb29",
+    "imageURL": "https://miro.medium.com/v2/resize:fit:1400/format:webp/1*9Z-1w5Qu9MoVLlhia94lQw.png"
+  },
+  "post2": {
+    "title": "Explaining content design to non-UXers",
+    "author": "Allison Wolfe",
+    "length": "3 min read",
+    "URL": "https://medium.com/patternfly/explaining-content-design-to-non-uxers-ab0779229a5c",
+    "imageURL": "https://miro.medium.com/v2/resize:fit:1400/0*Gy14UJRJ-QyF4c0C"
+  },
+  "post3": {
+    "title": "Critical skills for exceptional UX work",
+    "author": "Erin Donehoo",
+    "length": "4 min read",
+    "URL": "https://medium.com/patternfly/critical-skills-for-exceptional-ux-work-22cd57c264b3",
+    "imageURL": "https://miro.medium.com/v2/resize:fit:1400/0*34QuYnldEksEfyAI"
```

Comment:
This image is 1.2mb 😲 As a more of a thumbnail on our homepage, I think we should address that. The other images are grabbed from their articles, and bigger than they need to be, but in the 100-200k range. It would be great if we could optimize them, but it's not a huge deal.

We may also try modifying the URL here - it looks like they have a dynamic resize tool built into their site based on the URL. The max width we're displaying these images is around 600px, so this URL could be called via https://miro.medium.com/v2/resize:fit:600/0*34QuYnldEksEfyAI and it creates a much smaller image.

I suppose it's worth considering if we want to hot-link these images, too, or host optimized/smaller versions of them ourselves.

Reply:
Thanks @mcoker , that's a neat miro feature. Updated these URLs to include `resize:fit:600`